libs/libulfius/Makefile

```diff
@@ -2,7 +2,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libulfius
 PKG_VERSION:=2.7.10
```

Comment:
There is a new version available: `2.7.15`

Reply:
I saw that! I can include a version bump in this PR as well if you'd like.

Reply:
That would be great! :)
I'll appreciate every contribution as I currently have some time restrictions.